cvdupdate/cvdupdate.py

```diff
@@ -255,12 +255,12 @@ def update(self, db: str = "") -> bool:
 
     def clean_dbs(self):
         """
-        Delete all files in the database directory.
+        Delete cvd controlled files in the database directory.
         """
-        self.logger.info(f"Deleting databases...")
-        dbs = self.db_dir.glob('*')
+        dbs = self.config['dbs'].keys()
         for db in dbs:
-            os.remove(str(db))
+            cvddb = str(self.db_dir) + "/" + db
```

Comment:
Instead of hardcoding a `/` path separator, it would be good to change this to something like `str(self.db_dir / db)`, where the `/` division operator works with pathlib's Path() objects to use the appropriate unix or windows path separator.